build/build-image/cross/Dockerfile

```diff
@@ -72,7 +72,7 @@ RUN mkdir $TMPDIR \
             github.com/jteeuwen/go-bindata/go-bindata
 
 # Download and symlink etcd. We need this for our integration tests.
-RUN export ETCD_VERSION=v2.2.1; \
+RUN export ETCD_VERSION=v3.0.3; \
```

Comment:
Ideally yes, but we'd need the container pushed to google repos, I can't make that happen through the PR process.